platform-core-module/src/main/java/com/softicar/platform/core/module/event/AGSystemEvent.sqml

```diff
@@ -0,0 +1,14 @@
+PACKAGE com.softicar.platform.core.module.event
+
+IMPORT com.softicar.platform.core.module.event.severity.AGSystemEventSeverity
+IMPORT com.softicar.platform.core.module.user.AGUser
+
+TABLE AGSystemEvent "Core.SystemEvent" {
+	PK Integer id = SERIAL
+	Boolean needsAttention
+	AGSystemEventSeverity severity
+	AGUser causedBy
+	DayTime causedAt
```

Comment:
I think we should swap `causedAt` and `causedBy`.
The sequence in which things happened (as denoted via `causedAt`) is usually more important than finding out who caused them (as denoted via `causedBy`). Or, the other way around: In absence of temporal ordering of events, the "who" would be almost irrelevant.
The column order in `AGTransaction` might have emerged from the same thought.